packages/legend-studio/src/stores/editor-state/GraphGenerationState.ts

```diff
@@ -130,7 +131,8 @@ export class GraphGenerationState {
       const currentElement = this.editorStore.currentEditorState.element;
       if (
         currentElement instanceof Class ||
-        currentElement instanceof Enumeration
+        currentElement instanceof Enumeration ||
+        currentElement instanceof ConcreteFunctionDefinition
```

Comment:
Let's also make extension mechanism out of this

Reply:
I suspect we don't have specific FileGeneratoinSpecification at this point, thus cannot know the generation type (if it's Morphir), which we need for the extension. Should we revisit this?

Reply:
Added extension filter for generation object type scope and implemented case for Morphir.